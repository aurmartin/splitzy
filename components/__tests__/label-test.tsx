import * as React from "react";
import { render } from "@testing-library/react-native";
import Label from "../label";

describe("Label", () => {
  it("renders correctly", () => {
    const tree = render(<Label>Test Label</Label>);
    expect(tree).toMatchSnapshot();
  });
});
