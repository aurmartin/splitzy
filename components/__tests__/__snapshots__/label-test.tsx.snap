// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label renders correctly 1`] = `
<Text
  style={
    [
      {
        "fontSize": 16,
        "fontWeight": "400",
        "lineHeight": 24,
      },
      {
        "marginBottom": 4,
      },
    ]
  }
>
  Test Label
</Text>
`;
